### PR TITLE
Jak/Stat example as additional test case

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: tests/cpputest/wrapTestModels.m [hunk 1]
```diff
@@ -4,20 +4,38 @@
...
-
-    [exdir,~,~]=fileparts(which('example_steadystate.m'));
-    amiwrap('model_steadystate','model_steadystate_syms',exdir)
-          
+    
+    try
+        [exdir,~,~]=fileparts(which('example_steadystate.m'));
+        amiwrap('model_steadystate','model_steadystate_syms',exdir);
+    catch
+        cd(fileparts(mfilename('fullpath')));
```
Comment (FFroehlich) [inline tests/cpputest/wrapTestModels.m]: don't you want to compile again after cd?

Comment (dweindl) [inline tests/cpputest/wrapTestModels.m]: The 'catch' was only meant to change back to the original working directory, no real error handling is taking place. The problem throwing the exception in the first place will probably persist.

Comment (FFroehlich) [inline tests/cpputest/wrapTestModels.m]: ah okay!

**Comment by dweindl** [comment]
Tests are failing on osx due to numerical errors. What would be a reasonable tolerance (currently 1e-8)?
`numstepsS` and `numrhsevalsS` also differ. Does it make sense to check those?

**Comment by FFroehlich** [comment]
numrhsevals(S) and numsteps(S) may differ, so I would not test for them. as far as I can tell you are using default integration tolerances, which means that rtol is 1e-8, as I would expect a loss of 2 orders of magnitude and the problematic value is ~0.2 so an error of magnitude 1e-7 is still okayish.
Is it possible to check both absolute and relative error and only throw an error if one of the two exceeds an threshold?

**Comment by FFroehlich** [comment]
Alternatively set rtol to 1e-10, an absolute accuracy of 1e-8 is something that we should aim for

**Comment by dweindl** [comment]
Okay, I will remove numrhsevals(S) and numsteps(S).

Checking for relative accuracy makes more sense indeed. Thresholds for both relative and absolute can be implemented, yes.

**Comment by FFroehlich** [comment]
Okay but keep in mind that **either** relative or absolute tolerances must hold, not necessarily both.

File: tests/cpputest/jakstat_adjoint/tests1.cpp [hunk 2]
```diff
@@ -6,6 +6,7 @@
 #include <cstring>
 #include "wrapfunctions.h"
 
+#define TEST_RTOL_JAKSTAT 1e-06
```
Comment (FFroehlich) [inline tests/cpputest/jakstat_adjoint/tests1.cpp]: does not need to be jakstat specific, with rtol 1e-8, I would not expect more than 1e-6 for any of the models

**Pull request merged**